misc/config_tools/schema/config.xsd

```diff
@@ -104,6 +104,13 @@ multiboot2 protocol. If set to ``n``, GRUB's multiboot2 is not available as a
 boot option.</xs:documentation>
       </xs:annotation>
     </xs:element>
+    <xs:element name="ENFORCE_TURNOFF_AC" type="Boolean" default="n">
+      <xs:annotation>
+        <xs:documentation>Force to disable #AC for Split-locked Access.If CPU has #AC for
+split-locked access, HV enable it and VMs can't disable. Set this to enforce turn off that
+#AC, for community developer only.</xs:documentation>
```

Comment:
`enable` -> `enables`
`enforce turn off` -> `enforce turning off`
`for community developer only` -> do you mean `for debugging purposes only`?

Reply:
updated